modules/oe_theme_content_event/src/Plugin/ExtraField/Display/ProgrammeExtraField.php

```diff
@@ -113,10 +113,13 @@ public function getLabel() {
 
   /**
    * {@inheritdoc}
+   *
+   * @SuppressWarnings(PHPMD.CyclomaticComplexity)
+   * @SuppressWarnings(PHPMD.NPathComplexity)
    */
   public function viewElements(ContentEntityInterface $entity) {
     if ($entity->get('oe_event_programme')->isEmpty()) {
-      return;
+      return [];
```

Comment:
We shouldn't return empty array but use the `$isEmpty` flag of the `ExtraFieldDisplayFormattedBase`.